### DISCUSSION TITLE
[Request] Google Fit API Supported (Non-Brands as not supporting hyperate/pulsroid)

### DISCUSSION THREAD

**Discussion by NekoSuneVR** [VolcanicArts/VRCOSC#88] [Ideas]
would like see integration of Google Fit API pull Heart Rate system, it might require 0Auth2 in localhost Connection and store they Google Access Token Localy for Security reasons.

since some non-brands doesn't support hyperate or pulsroid platforms and only has supported is google fit Relay integrations including fitbit as well. 

i have heart rate watch that only supports google fit. if able make this feature be useful for most watches for Android Users.

**Comment by VolcanicArts** [comment]
I'll add it to the to-do list and see what I can do
